Task 8/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение x для функции y=f(x)");
+            var x = double.Parse(Console.ReadLine());
+
+            Console.WriteLine($"f({x}) = {MyFunction(x)}");
+            Console.ReadKey();
+        }
+            static double MyFunction(double x)
+            {
+                if (x < -1)
+                    return 1;
+                else if (x > 1)
+                    return -1;
+                else 
+                    return (x*(-1));
```

Comment:
Лишние скобки затрудняют чтение кода. Вместо умножения проще использовать смену знака
```suggestion
                    return -x;
```